h2o-docs/src/product/data-science/if.rst

```diff
@@ -221,6 +221,12 @@ Below is a simple example showing how to build an Isolation Forest model.
         # Predict the leaf node assignment
         ln_pred = model.predict_leaf_node_assignment(test, "Path")
 
+FAQ
+~~~
+
+- **How does the algorithm handle missing values during training?**
+
+    When correctly imported, Isolation Forest handles missing values the same way as DRF. Missing values are interpreted as containing information (as opposed to missing information at random). Missing values are counted during splits when splitting for Feature and Value. They can either go to the left or right branch.
```

Comment:
` Missing values are interpreted as containing information (as opposed to missing information at random).`

I don't understand to this sentence. Missing values are just missing values. I am not familiar with any thing like `missing information at random`

I think if we left out the sentence then it is more clear.

` When correctly imported, Isolation Forest handles missing values the same way as DRF. Missing values are counted during splits when splitting for Feature and Value. They can either go to the left or right branch.
`

Reply:
@valenad1 Since IF handles missing values like DRF, I looked at [the section on the DRF page that describes how it handles missing values](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/drf.html#faq). That's where I got the `Missing values are interpreted as containing information (as opposed to missing information at random)` sentence. 

Since it doesn't make sense, I'm going to update the DRF FAQ as well and pull that sentence 👍

Reply:
Ahh ok, now I understand, my bad.. this make sense.. IMHO the `(i.e., missing for a reason)` for understanding :)